### ISSUE TITLE
Create a "Join" page to provide instructions for how a package can join E4S

### ISSUE THREAD

**Issue by maherou** [E4S-Project/E4S-Project.github.io#9]
We need a page on e4s-project.github.io that describes how a product can join E4S.  The page should also have a Join button at the top of pages, perhaps between "Contact Us" and "FAQ" or perhaps we could refactor one of the top buttons (such as "Contact Us") to include a link to the Join page.

The Join page should have the following information:

Pre-requisite: Justification for being included.  The product must have some value or strong potential value to the HPC community.  In other words, it must make sense for the product to belong to E4S. **Note: It probably makes sense to have this pre-requisite be a new E4S community policy. See issue #8.**

Level 0: Be listed in the E4S Spack installation script and be buildable in all E4S target environments.  The list of Spack recipes for E4S is here: https://github.com/UO-OACISS/e4s/tree/master/e4s-facility-environments/uoregon.

Level 1: Be present in the DocPortal: https://e4s-project.github.io/DocPortal.html. To achieve this, we need a URL to your main repo to add to this list: https://github.com/E4S-Project/E4S-Documenter/blob/master/data/e4s_products.yaml.

Level 3: Satisfy all E4S community policies: https://e4s-project.github.io/policies.html. Full satisfaction is not required at this time.  We are still establishing a process.  Note that derived requirements from E4S member packages are not required to satisfy community policies as long as they do destabilize E4S builds or portability.

**Comment by wspear** [comment]
We have a first run at a Join link in place: https://e4s-project.github.io/join.html

**Comment by jedbrown** [comment]
This remains obtuse in that it just goes to the generic "contact us" page (presumably this is an email to the technical lead) and the Level 0/1/2 don't define what should be done by the submitter (they can't unilaterally add to these registries) before emailing. A form or GitHub issue template would streamline that.

**Comment by maherou** [comment]
@jedbrown Agreed.  We are working on an extensive revision of the E4S site.  Thanks.
